scripts/ansible/roles/avalanche_build/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
-avalanche_binary: ~/go/src/github.com/ava-labs/avalanchego/build/avalanchego
-repo_folder: ~/go/src/github.com/ava-labs/avalanchego
+avalanche_binary: "{{ repo_folder }}/build/avalanchego"
+repo_folder: "~{{ ansible_user }}/go/src/github.com/ava-labs/avalanchego"
```

Comment:
Where is the `ansible_user` variable set and what value does it have? Is there a default?

I don't see this set anywhere in our codebase, would this need an entry in the `vars/main.yml` file?